src/pages/colinks/explore/SimpleBuyButtonWithPrice.tsx

```diff
@@ -33,7 +35,7 @@ export const SimpleBuyButtonWithPrice = ({
       <BuyButton
         setProgress={setProgress}
         onSuccess={onSuccess}
-        target={target}
+        target={target as Address}
```

Comment:
don't love these `as` moves throughout maybe there is a better way

Reply:
refactor all components to take and pass Address types @CryptoGraffe